ansible/roles/common/tasks/main.yml

```diff
@@ -17,70 +17,39 @@
   apt:
     update_cache: yes
 
-- name: install librados-dev and librbd-dev external packages
+- name: install make, gcc, librados-dev and librbd-dev external packages
```

Comment:
Add git and go?

Reply:
Since I only test the environment on Ubuntu where `git` is naturally included, so there is no need to add it. As for go, I think kubernetes already requires for go in local cluster, it would work for opensds.